packages/nextjs/components/HooksExample.tsx

```diff
@@ -0,0 +1,70 @@
+import { ReactNode } from "react";
+
+const Red = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#FF5962]">{children}</span>;
+};
+const Cyan = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#00B0A6]">{children}</span>;
+};
+
+const Purple = ({ children }: { children: ReactNode }) => {
+  return <span className="text-[#7B61FF]">{children}</span>;
+};
+export const HooksExample = () => {
+  return (
+    <div className="w-full flex-grow  bg-white rounded-2xl p-6 drop-shadow-[0px_0px_10px_rgba(139,178,241,0.50)]">
+      <div className="bg-base-300/40 w-full h-full border border-primary rounded-3xl p-4 lg:p-6 text-xs lg:text-sm  font-mono overflow-x-scroll whitespace-nowrap  lg:overflow-auto lg:whitespace-normal">
+        <p className="my-3">
+          <Red>import</Red>
+          {" { "} useScaffoldContractWrite {" } "}
+          <Red>from</Red> <Cyan>&quot; ~~/hooks/scaffold-eth &quot;</Cyan>;
+        </p>
+
+        <p className="my-3">
+          <Red>const</Red>
+          {" { "}writeAsync, isLoading, isMining{" } "}
+          <Cyan> = </Cyan>
+          <br />
+          <Purple>useScaffoldContractWrite</Purple>({"{"}
+          <br />
+          <span className="inline-block ml-4">
+            <Cyan>contractName</Cyan>: <Cyan> &quot;YourContract&quot;</Cyan>,
```

Comment:
One question, should `contractName:`, `functionName:`, `args:` and `blockConfirmation:` have a different color than their values? (like black for example?). 

Maybe is just a bias from my VSC coloring.

Reply:
Yup, actually Figma had the values in a bit darker : 

![Screenshot 2023-10-18 at 12 44 21 PM](https://github.com/scaffold-eth/scaffoldeth.io/assets/80153681/34097696-abd0-4683-bacc-6518b9b1b050)

I updated it the same as Figma, like it differentiates them a bit but also looks simple and clean

![Screenshot 2023-10-18 at 12 53 31 PM](https://github.com/scaffold-eth/scaffoldeth.io/assets/80153681/acfdce85-1e87-48d9-a04c-2d36b59dff19)

even [tokyonight theme](https://github.com/folke/tokyonight.nvim) does it similarly were keys are dark and values are light

---

I don't know if having a different color will make it more overpowered, this is how it looks with black : 

![Screenshot 2023-10-18 at 11 45 46 AM](https://github.com/scaffold-eth/scaffoldeth.io/assets/80153681/08cacfce-d7c4-4e7f-ad02-d8186d6212f5)

Reply:
I think is cool as you've set it 🙌